src/KeyHandler.h

```diff
@@ -76,6 +76,10 @@ class KeyHandler {
       const InputStates::ChoosingCandidate::Candidate& candidate,
       const StateCallback& stateCallback);
 
+  void candidateAssociatedPhraseSelected(
```

Comment:
Due to the issue discovered in https://github.com/openvanilla/fcitx5-mcbopomofo/pull/108, please make the callback parameter `StateCallback stateCallback` instead of `const StateCallback& stateCallback)`.